bucket/rescuetime.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "2.14.5.3",
+    "description": "Automatic application time-tracking and productive advisor",
+    "homepage": "https://www.rescuetime.com",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.rescuetime.com/tos"
+    },
+    "url": "https://www.rescuetime.com/installers/RescueTimeInstaller_2.14.5.3.exe",
+    "hash": "abe2005aa5a2fa700ca3faf88c6ed36638448427c202af1a716094639601e0bd",
+    "innosetup": true,
+    "bin": "RescueTime.exe",
+    "shortcuts": [
+        [
+            "RescueTime.exe",
+            "RescueTime"
+        ]
+    ],
+    "persist": "options.txt",
+    "checkver": {
+        "url": "https://www.rescuetime.com/updates/win_release_notes.html",
+        "regex": "<strong>([\\d.]+)</strong>"
+    },
+    "autoupdate": {
+        "url": "https://www.rescuetime.com/installers/RescueTimeInstaller_$version.exe"
+    },
+    "notes": [
```

Comment:
Remove all of this (or just link to specific browsers). It is nonsense to keep it here. 

Also I have no idea why did you not mention brave...

Reply:
**You probably meant this (why remove the autoupdater?):**

https://github.com/lukesampson/scoop-extras/blob/8da730a73987a60e01434ba784e301a21c7afdfd/bucket/rescuetime.json#L27-L32

> Remove all of this (or just link to specific browsers). It is nonsense to keep it here.

I'm struggling to understand what you actually mean with "just link to specific browsers", do I remove the links from `notes` or do I remove the message on the top?

> Also I have no idea why did you not mention brave...

We're talking about extensions, not supported browsers (I guess you got the Brave reference from the `checkver` link). The chrome extension works for Brave (and the new Edge, aswell, because the three come all from the same Chromium base).

I could rename that to "Chrome/Brave/Edge" (the new ms edge is announced to be released in a week or so)...

Reply:
Remove links

Reply:
¯\_(ツ)_/¯